### PR TITLE
examples: Add P, S -velocity and density parametrization option based on C matrix and gradient vector 

### PR DESCRIPTION
**Pull request by guaacoelho**

### CODE REVIEW

File: examples/seismic/stiffness/operators.py [hunk 1]
```diff
@@ -96,7 +97,59 @@ def elastic_stencil(model, v, tau, forward=True):
...
+    hvp = TimeFunction(name='hvp', grid=model.grid, space_order=space_order,
+                           time_order=1)
+    hvs = TimeFunction(name='hvs', grid=model.grid, space_order=space_order,
+                           time_order=1)
+    hr = TimeFunction(name='hr', grid=model.grid, space_order=space_order,
+                           time_order=1)
+
+    Wvp = gather(0, C.dvp * S(v))
+    Wvs = gather(0, C.dvs * S(v))
+    Wr = gather(0, C.drho * S(v))
```
Comment (nogueirapeterson) [inline examples/seismic/stiffness/operators.py:136]: Wr = gather(v.dt, - C.drho * S(v))

change "0" by "v.dt" and put "-" in "C.drho"

File: examples/seismic/stiffness/operators.py [hunk 2]
```diff
@@ -96,7 +97,59 @@ def elastic_stencil(model, v, tau, forward=True):
...
+    W2 = gather(u, sig)
+
+    wvp_update = Eq(hvp , Wvp.T * W2)
+    gradient_lam= Eq(grad_vp, grad_vp + hvp)
+
+    wvs_update = Eq(hvs , Wvs.T * W2)
+    gradient_mu = Eq(grad_vs, grad_vs + hvs)
+
+    wr_update = Eq(hr , Wr.T * W2)
+    gradient_rho = Eq(grad_rho, grad_rho + hr)
```
Comment (nogueirapeterson) [inline examples/seismic/stiffness/operators.py:147]: gradient_rho = Eq(grad_rho, grad_rho - hr)

put "-" in "hr"

**Review by nogueirapeterson — CHANGES_REQUESTED** [review]

**Review by nogueirapeterson — APPROVED** [review]

**Review by AtilaSaraiva — COMMENTED** [review]
Just as a note for future PRs, these commits have way too many lines, its kinda hard to follow the history of the changes when there are so many lines changed in each commit.

**Comment by nogueirapeterson** [comment]
> Just as a note for future PRs, these commits have way too many lines, its kinda hard to follow the history of the changes when there are so many lines changed in each commit.

@guaacoelho
